commands/spoiler.js

```diff
@@ -0,0 +1,14 @@
+module.exports.execute = async (client, message, args) => {
+
+	let spoilerText = '||' + args.join(' ') + '||';
```

Comment:
Would you able to handle the case in which no arguments are provided? Something like the `!apply` command, a quick reminder of the intended usage

Reply:
Thanks for the tip! I'll add that.